main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: mana
+//  action 1: check amount of mana
+mana.checkAmount();
+//  action 2: use mana
+mana.useMana();
+//  action 3: refill mana
+mana.refillMana();

```

Comment:
This noun doesn't perform these actions.
Something else (a different noun) performs these actions TO or WITH this noun.
- [ ] revise